Source/HelixToolkit.Wpf.SharpDX/Controls/Viewport3DX.cs

```diff
@@ -1001,69 +1006,92 @@ protected virtual void OnCameraChanged()
         }
 
         /// <summary>
-        /// Invoked when an unhandled MouseUp attached event reaches an element in its route that is derived from this class. Implement this method to add class handling for this event.
+        /// Gets the pressed mouse buttons as flags of <see cref="MouseButton"/>.
+        /// If not button is pressed (result is zero), then it was a touch down.
         /// </summary>
-        /// <param name="e"></param>
-        protected override void OnMouseUp(MouseButtonEventArgs e)
+        /// <returns>
+        /// The pressed mouse buttons as flags of <see cref="MouseButton"/>.
+        /// </returns>
+        public static MouseButton GetPressedMouseButtons()
         {
-            base.OnMouseUp(e);
-            this.MouseUpHitTest(e);
+            int flags = 0;
+            flags |= (int)Mouse.LeftButton << 0;
+            flags |= (int)Mouse.MiddleButton << 1;
+            flags |= (int)Mouse.RightButton << 2;
+            flags |= (int)Mouse.XButton1 << 3;
+            flags |= (int)Mouse.XButton2 << 4;
+            return (MouseButton)flags;
         }
 
-        public void EmulateMouseUpByTouch(MouseButtonEventArgs e)
+        /// <inheritdoc/>

```

Comment:
I voted for the feature at https://visualstudio.uservoice.com/forums/121579-visual-studio/suggestions/3745102-add-intellisense-support-for-the-inheritdoc-ta